wireless_modules/mqtt_test/config.example.py

```diff
@@ -0,0 +1,4 @@
+ESSID = "<WIFI NETWORK NAME HERE>"
+PASSWORD = "<WIFI PASSWORD HERE>"
+
+MQTT_BROKER = "192.168.1.111"
```

Comment:
Any reason for this IP address?

Reply:
It was just the one I was using at home. What is the DAS IP? I'll change it to the DAS one

Reply:
For V2 we used `192.168.100.100`. I'd really like to use a thing called mDNS though, cos then we could use something like `das.local`. But that IP should be good for now